returnn/frontend_api.py

```diff
@@ -42,6 +42,20 @@ def convert_to_tensor(value: Union[Tensor, T, RawTensorTypes]) -> Tensor[T]:
         """
         raise NotImplementedError
 
+    @staticmethod
+    def _tensor_op_sanity_check(t1: Union[RawTensorTypes, Tensor], t2: Union[RawTensorTypes, Tensor]):
+        """
+        Performs basic checks common to all _TensorMixin binary operations,
+        like _TensorMixin.__eq__ or _TensorMixin.__add__.
```

Comment:
There should be an empty line after the descriptions, before the params. (Keep consistency.)